### PR TITLE
Added Ground_Block and Ground_Event to eDSL

### LINKED ISSUE #641: Add SeqJSON Spec GroundBlock type to Step (2 of 5)
We want to connect the `GroundBlock` Interface of the SeqJson Spec to the existing `CommandStem` eDSL Class for the `step` object in the `Sequence` Class.

```ts
Sequence.new({
   seqId: '',
   metadata: {},
   steps: [ GroundBlock ]
 });
```

### LINKED ISSUE #642: Add SeqJSON Spec GroundEvent type to Step (3 of 5)
We want to connect the `GroundEvent` Interface of the SeqJson Spec to the existing `CommandStem` eDSL Class for the `step` object in the `Sequence` Class.

```ts
Sequence.new({
   seqId: '',
   metadata: {},
   steps: [ GroundEvent ]
 });
```

### PR DESCRIPTION
**Pull request by camargo**
Closes https://github.com/NASA-AMMOS/aerie/issues/641 
Closes https://github.com/NASA-AMMOS/aerie/issues/642

Original PR: https://github.com/NASA-AMMOS/aerie/pull/700
We needed to [revert](https://github.com/NASA-AMMOS/aerie/pull/713) because creating an expansion set fails.

### CODE REVIEW

**Comment by camargo** [comment]
Here is the error we get when creating an expansion set with this branch:

```
[node] Error: Expansion set could not be type checked
[node] 	Caused by: [
[node] 		TypeError: TS2322 Type '{ [x: string]: ((...args: any[]) => any) | ((...args: any[]) => any); }' is not assignable to type '{ FSW_CMD_0: (args_0: { enum_arg_0: "ON" | "OFF"; boolean_arg_0: boolean; float_arg_0: number; }) => FSW_CMD_0; FSW_CMD_1: (args_0: { float_arg_0: number; integer_arg_0: number; time_arg_0: string; unsigned_arg_0: number; var_string_arg_0: string; }) => FSW_CMD_1; } & { ...; }'.
[node] 		  Type '{ [x: string]: ((...args: any[]) => any) | ((...args: any[]) => any); }' is missing the following properties from type '{ FSW_CMD_0: (args_0: { enum_arg_0: "ON" | "OFF"; boolean_arg_0: boolean; float_arg_0: number; }) => FSW_CMD_0; FSW_CMD_1: (args_0: { float_arg_0: number; integer_arg_0: number; time_arg_0: string; unsigned_arg_0: number; var_string_arg_0: string; }) => FSW_CMD_1; }': FSW_CMD_0, FSW_CMD_1
[node] 	]
[node] 		at file:///app/build/app.js:181:15
[node] 	Caused by: [
[node] 		at (15:45139)
[node] 	]
```

Using this expansion rule on Banananation, targeting `PeelBanana`:

```ts
export default function MyExpansion(props: { activityInstance: ActivityType }): ExpansionReturn {
  const { activityInstance } = props;
  const { peelDirection } = activityInstance.attributes.arguments;

  return [
    C.FSW_CMD_1({
      float_arg_0: 0,
      integer_arg_0: 0,
      time_arg_0: '00:00:00',
      unsigned_arg_0: 0,
      var_string_arg_0: peelDirection,
    }),
  ];
}
```

Using this command dictionary: https://github.com/NASA-AMMOS/aerie-ui/blob/develop/e2e-tests/data/command-dictionary.xml

File: sequencing-server/test/sequence-generation.spec.ts [hunk 1]
```diff
@@ -2587,7 +2587,7 @@ describe('sequence generation', () => {
   }, 30000);
 });
 
-describe('expansion', () => {
+describe('expansion regressions', () => {
```
Comment (camargo) [inline sequencing-server/test/sequence-generation.spec.ts:2590]: We should add a test here or in the e2e-tests that creates an expansion rule, creates an expansion set, and runs expansion.

Comment (cohansen) [reply sequencing-server/test/sequence-generation.spec.ts:2590]: I added a test on line `143` that creates an expansion set to test when there's a name conflict in the command dictionary and model, it's interesting that isn't catching this new error.

Comment (camargo) [reply sequencing-server/test/sequence-generation.spec.ts:2590]: So I'm testing this a little bit more this morning. It seems to be [this command dictionary](https://github.com/NASA-AMMOS/aerie-ui/blob/develop/e2e-tests/data/command-dictionary.xml) that is causing the error. The [command dictionary in the sequencing server](https://github.com/NASA-AMMOS/aerie/blob/develop/sequencing-server/cdict/command_banananation.xml) does not cause the error.

Comment (goetzrrGit) [reply sequencing-server/test/sequence-generation.spec.ts:2590]: @camargo this test does create an expansion rule, set and run expansions. We do it multiple times in this test.

**Comment by goetzrrGit** [comment]
@camargo it should be fixed now

**Review by cohansen — APPROVED** [review]
Looks good, tests passed on my end.

**Pull request merged**